### PR TITLE
Update extending-messaging-with-integrations.md

### PR DESCRIPTION
**Pull request by justinegeffen**
Added integrations sections/details.

### CODE REVIEW

File: source/messaging/extending-messaging-with-integrations.md [hunk 1]
```diff
@@ -1,9 +1,111 @@
...
+Settings for integrations are accessible from the **Main Menu**. Select **Integrations** to open a page where you can view and configure incoming webhooks, outgoing webhooks, and slash commands for your team. If you can't see an **Integrations** option, then your System Admin may have only given Admins access.
 
 [Visit our app directory](https://about.mattermost.com/default-app-directory/) for dozens of open source integrations to common tools like Jira, Jenkins, GitLab, Trac, Redmine, and Bitbucket, along with interactive bot applications (Hubot, mattermost-bot), and other communication tools (Email, IRC, XMPP, Threema) that are freely available for use and customization.
 
+### Microsoft integrations
+
+There are several ways to connect your Microsoft tools to Mattermost. Take a look at the tools recommended below, or visit the Marketplace to find others.
```
Comment (cwarnermm) [inline source/messaging/extending-messaging-with-integrations.md:9]: Consider including a link here to the new Marketplace.

File: source/messaging/extending-messaging-with-integrations.md [hunk 2]
```diff
@@ -1,9 +1,111 @@
...
+
+#### Office 365 Calendar (E20)
+
+- Two-way integration between Mattermost and Office 365 Calendar, developed by Mattermost.
+- Receive a daily summary of calendar events, and accept or decline new events.
+- Reflect user status as "Do Not Disturb" when in a meeting scheduled via Outlook Calendar.
+- Docs: https://mattermost.gitbook.io/plugin-mscalendar
```
Comment (cwarnermm) [inline source/messaging/extending-messaging-with-integrations.md:16]:
```suggestion
- Documentation: https://mattermost.gitbook.io/plugin-mscalendar
```

File: source/messaging/extending-messaging-with-integrations.md [hunk 3]
```diff
@@ -1,9 +1,111 @@
...
+
+#### Skype for Business
+
+- Start and join voice calls, video calls, and use screen-sharing with your team members. Developed by kosgrz and maintained by Mattermost.
```
Comment (cwarnermm) [inline source/messaging/extending-messaging-with-integrations.md:20]: It looks like we're using the developer's GitHub handle? If so, does it make sense to link to his GitHub account as well?

File: source/messaging/extending-messaging-with-integrations.md [hunk 4]
```diff
@@ -1,9 +1,111 @@
...
+- Clicking a video icon in a Mattermost channel invites team members to join a Skype for Business call, hosted using the credentials of the user who initiated the call.
+- Supports a vendor-hosted cloud solution.
+- Source code + docs: https://github.com/mattermost/mattermost-plugin-skype4business
```
Comment (cwarnermm) [inline source/messaging/extending-messaging-with-integrations.md:23]:
```suggestion
- Source code + documentation: https://github.com/mattermost/mattermost-plugin-skype4business
```

File: source/messaging/extending-messaging-with-integrations.md [hunk 5]
```diff
@@ -1,9 +1,111 @@
...
+
+#### Office 365 SSO (E20)
+
+- Configure Mattermost to use your Office 365 credentials and Azure Active Directory account as a single sign-on (SSO) service, developed by Mattermost.
+- Supports Microsoft Active Directory Tenants for team creation, account creation, and SSO.
```
Comment (cwarnermm) [inline source/messaging/extending-messaging-with-integrations.md:28]:
```suggestion
- Supports Microsoft Active Directory Tenants for team creation, account creation, and Single Sign-On (SSO).
```

File: source/messaging/extending-messaging-with-integrations.md [hunk 6]
```diff
@@ -1,9 +1,111 @@
...
+- Receive a daily summary of calendar events, and accept or decline new events.
+- Docs: https://docs.mattermost.com/deployment/sso-office.html
```
Comment (cwarnermm) [inline source/messaging/extending-messaging-with-integrations.md:30]:
```suggestion
- Documentation: https://docs.mattermost.com/onboard/sso-office.html
```

File: source/messaging/extending-messaging-with-integrations.md [hunk 7]
```diff
@@ -1,9 +1,111 @@
...
+
+### CI/CD integrations
+
+There are several ways to connect your CI/CD tools to Mattermost. Take a look at the tools recommended below, or visit the Marketplace to find others.
```
Comment (cwarnermm) [inline source/messaging/extending-messaging-with-integrations.md:34]: Same comment as above - link to Marketplace

File: source/messaging/extending-messaging-with-integrations.md [hunk 8]
```diff
@@ -1,9 +1,111 @@
...
+
+#### GitHub
+
+ - Two-way integration between Mattermost and GitHub, developed by Mattermost.
+ - Get reminders on issues and pull requests that need your attention.
+ - Get notifications in Mattermost about mentions, review requests, and comments.
+ - Supports GitHub Enterprise, works with SaaS and Enterprise versions of GitHub.
+ - Source code + docs: https://github.com/mattermost/mattermost-plugin-github
```
Comment (cwarnermm) [inline source/messaging/extending-messaging-with-integrations.md:42]:
```suggestion
 - Source code + documentation: https://github.com/mattermost/mattermost-plugin-github 
```

File: source/messaging/extending-messaging-with-integrations.md [hunk 9]
```diff
@@ -1,9 +1,111 @@
...
+
+ - Two-way integration between Mattermost and GitHub, developed by Mattermost.
+ - Get reminders on issues and pull requests that need your attention.
+ - Get notifications in Mattermost about mentions, review requests, and comments.
+ - Supports GitHub Enterprise, works with SaaS and Enterprise versions of GitHub.
+ - Source code + docs: https://github.com/mattermost/mattermost-plugin-github 
+
+#### GitLab
+
+ - Two-way integration between Mattermost and GitLab, developed by Romain Maneschi, supported by Mattermost.
```
Comment (cwarnermm) [inline source/messaging/extending-messaging-with-integrations.md:46]: Same comment as above re: developer's handle (if possible)

File: source/messaging/extending-messaging-with-integrations.md [hunk 10]
```diff
@@ -1,9 +1,111 @@
...
+ - Get reminders on issues and merge requests that need your attention.
+ - Get notifications in Mattermost about mentions, review requests, and comments.
+ - Supports SaaS and on-prem versions of GitLab.
+ - Source code + docs: https://github.com/mattermost/mattermost-plugin-gitlab
```
Comment (cwarnermm) [inline source/messaging/extending-messaging-with-integrations.md:50]:
```suggestion
 - Source code + documentation: https://github.com/mattermost/mattermost-plugin-gitlab
```

File: source/messaging/extending-messaging-with-integrations.md [hunk 11]
```diff
@@ -1,9 +1,111 @@
...
+
+### Atlassian integrations
+
+There are several ways to connect Atlassian tools to Mattermost. Take a look at the tools recommended below, or visit the Marketplace to find others.
```
Comment (cwarnermm) [inline source/messaging/extending-messaging-with-integrations.md:54]: Link to Marketplace?

File: source/messaging/extending-messaging-with-integrations.md [hunk 12]
```diff
@@ -1,9 +1,111 @@
...
+
+#### Confluence
+
+- Atlassian Marketplace application for Confluence and Mattermost, developed by codefortynine.
```
Comment (cwarnermm) [inline source/messaging/extending-messaging-with-integrations.md:58]: Developer link?

File: source/messaging/extending-messaging-with-integrations.md [hunk 13]
```diff
@@ -1,9 +1,111 @@
...
+- Send notifications about page, blogpost, question or comment updates from your Confluence spaces to Mattermost channels.
```
Comment (cwarnermm) [inline source/messaging/extending-messaging-with-integrations.md:59]:
```suggestion
- Send notifications about page, blogpost, question, or comment updates from your Confluence spaces to Mattermost channels.
```

File: source/messaging/extending-messaging-with-integrations.md [hunk 14]
```diff
@@ -1,9 +1,111 @@
...
+- Configure notifications for a specific space, and also user notifications for tasks and mentions.
+- Supports Confluence Cloud, Server and Data Center platforms.
```
Comment (cwarnermm) [inline source/messaging/extending-messaging-with-integrations.md:61]:
```suggestion
- Supports Confluence Cloud, Server, and Data Center platforms.
```

File: source/messaging/extending-messaging-with-integrations.md [hunk 15]
```diff
@@ -1,9 +1,111 @@
...
+- Marketplace download + docs: https://marketplace.atlassian.com/apps/1215055/slack-for-confluence
```
Comment (cwarnermm) [inline source/messaging/extending-messaging-with-integrations.md:62]:
```suggestion
- Marketplace download + documentation: https://marketplace.atlassian.com/apps/1215055/slack-for-confluence
```

Comment (cwarnermm) [reply source/messaging/extending-messaging-with-integrations.md:62]: Why is this plugin in our docs called "Slack for Confluence"?

File: source/messaging/extending-messaging-with-integrations.md [hunk 16]
```diff
@@ -1,9 +1,111 @@
...
+
+#### Jira Plugin
+
+- Two-way integration between Mattermost and Jira, developed by Mattermost.
+- Send event notifications from your Jira projects to Mattermost channels, with full JQL filtering support.
+- Create and transition Jira issues, and attach Mattermost messages to Jira in the Mattermost user interface.
+- Supports Jira Core and Jira Software products, for Cloud, Server and Data Center platforms.
```
Comment (cwarnermm) [inline source/messaging/extending-messaging-with-integrations.md:69]:
```suggestion
- Supports Jira Core and Jira Software products, for Cloud, Server, and Data Center platforms.
```

File: source/messaging/extending-messaging-with-integrations.md [hunk 17]
```diff
@@ -1,9 +1,111 @@
...
+- Source code + docs: https://github.com/mattermost/mattermost-plugin-jira
```
Comment (cwarnermm) [inline source/messaging/extending-messaging-with-integrations.md:70]:
```suggestion
- Source code + documentation: https://github.com/mattermost/mattermost-plugin-jira
```

File: source/messaging/extending-messaging-with-integrations.md [hunk 18]
```diff
@@ -1,9 +1,111 @@
...
+
+### Productivity integrations
+
+There are several ways to connect your productivity tools to Mattermost. Take a look at the tools recommended below, or visit the Marketplace to find others.
```
Comment (cwarnermm) [inline source/messaging/extending-messaging-with-integrations.md:74]: Marketplace link?

File: source/messaging/extending-messaging-with-integrations.md [hunk 19]
```diff
@@ -1,9 +1,111 @@
...
+
+#### Google Calendar
+
+ - Time management tool for Google Calendar, developed by Wasim Thabraze.
```
Comment (cwarnermm) [inline source/messaging/extending-messaging-with-integrations.md:78]: Developer link?

File: source/messaging/extending-messaging-with-integrations.md [hunk 20]
```diff
@@ -1,9 +1,111 @@
...
+ - Get reminders about appointments and meetings from a configured Google Calendar to any Mattermost channel.
+ - Source code + docs: https://github.com/waseem18/mattermost-plugin-google-calendar
```
Comment (cwarnermm) [inline source/messaging/extending-messaging-with-integrations.md:80]:
```suggestion
 - Source code + documentation: https://github.com/waseem18/mattermost-plugin-google-calendar
```

File: source/messaging/extending-messaging-with-integrations.md [hunk 21]
```diff
@@ -1,9 +1,111 @@
...
+
+#### Remind Plugin
+
+ - A Mattermost plugin that sets reminders for users and channels, developed by Scott Lee Davis.
```
Comment (cwarnermm) [inline source/messaging/extending-messaging-with-integrations.md:84]: Developer link?

File: source/messaging/extending-messaging-with-integrations.md [hunk 22]
```diff
@@ -1,9 +1,111 @@
...
+ - Set reminders for one-time or recurring events and tasks in any Mattermost channel.
+ - Source code + docs: https://github.com/scottleedavis/mattermost-plugin-remind
```
Comment (cwarnermm) [inline source/messaging/extending-messaging-with-integrations.md:86]:
```suggestion
 - Source code + documentation: https://github.com/scottleedavis/mattermost-plugin-remind
```

File: source/messaging/extending-messaging-with-integrations.md [hunk 23]
```diff
@@ -1,9 +1,111 @@
...
+
+### Voice, video, and screensharing integrations
+
+There are several ways to work with voice, video, and screensharing in Mattermost. Take a look at the tools recommended below, or visit the Marketplace to find others.
```
Comment (cwarnermm) [inline source/messaging/extending-messaging-with-integrations.md:90]: Marketplace link?

File: source/messaging/extending-messaging-with-integrations.md [hunk 24]
```diff
@@ -1,9 +1,111 @@
...
+
+#### Zoom
+
+- Start and join voice calls, video calls, and use screensharing with your team members, developed by Mattermost.
+- Click a video icon in a Mattermost channel to invite team members to join a Zoom call, hosted using the credentials of the user who initiated the call.
+- Requires one paid `Zoom Pro, Business, Education, or API plan, <https://zoom.us/pricing>`_ to generate an API key and secret.
```
Comment (cwarnermm) [inline source/messaging/extending-messaging-with-integrations.md:96]:
```suggestion
- Requires one paid `Zoom Pro, Business, Education, or API plan <https://zoom.us/pricing>`_ to generate an API key and secret.
```

File: source/messaging/extending-messaging-with-integrations.md [hunk 25]
```diff
@@ -1,9 +1,111 @@
...
+- Supports a self-hosted cloud solution and a vendor-hosted cloud solution.
+- Docs available at: https://docs.mattermost.com/integrations/zoom.html.
```
Comment (cwarnermm) [inline source/messaging/extending-messaging-with-integrations.md:98]:
```suggestion
- Documentation available at: https://mattermost.gitbook.io/plugin-zoom/.
```

File: source/messaging/extending-messaging-with-integrations.md [hunk 26]
```diff
@@ -1,9 +1,111 @@
...
+
+- Two-way integration between Mattermost and Office 365 Calendar, developed by Mattermost.
+- Receive a daily summary of calendar events, and accept or decline new events.
+- Reflect user status as "Do Not Disturb" when in a meeting scheduled via Outlook Calendar.
+- Docs: https://mattermost.gitbook.io/plugin-mscalendar
+
+#### Skype for Business
+
+- Start and join voice calls, video calls, and use screen-sharing with your team members. Developed by kosgrz and maintained by Mattermost.
+- Clicking a video icon in a Mattermost channel invites team members to join a Skype for Business call, hosted using the credentials of the user who initiated the call.
```
Comment (cwarnermm) [inline source/messaging/extending-messaging-with-integrations.md:21]:
```suggestion
- Selecting a video icon in a Mattermost channel invites team members to join a Skype for Business call, hosted using the credentials of the user who initiated the call.
```

File: source/messaging/extending-messaging-with-integrations.md [hunk 27]
```diff
@@ -1,9 +1,111 @@
...
+- Click a video icon in a Mattermost channel to invite team members to join a Zoom call, hosted using the credentials of the user who initiated the call.
+- Requires one paid `Zoom Pro, Business, Education, or API plan, <https://zoom.us/pricing>`_ to generate an API key and secret.
+- Supports a self-hosted cloud solution and a vendor-hosted cloud solution.
+- Docs available at: https://docs.mattermost.com/integrations/zoom.html.
+- Source code available at: https://github.com/mattermost/mattermost-plugin-zoom.
+
+#### Microsoft Teams Meetings
+
+- Start and join voice calls, video calls, and use screensharing with your team members, developed by Mattermost.
+- Click a video icon in a Mattermost channel to invite team members to join a Microsoft Teams Meeting, hosted using the credentials of the user who initiated the call.
```
Comment (cwarnermm) [inline source/messaging/extending-messaging-with-integrations.md:104]:
```suggestion
- Select a video icon in a Mattermost channel to invite team members to join a Microsoft Teams Meeting, hosted using the credentials of the user who initiated the call.
```

File: source/messaging/extending-messaging-with-integrations.md [hunk 28]
```diff
@@ -1,9 +1,111 @@
...
+- Requires a Microsoft Teams account.
+- Requires Mattermost Enterprise Edition E20.
+- Source code and docs available at: https://github.com/mattermost/mattermost-plugin-msteams-meetings.
```
Comment (cwarnermm) [inline source/messaging/extending-messaging-with-integrations.md:107]:
```suggestion
- Source code and documentation available at: https://github.com/mattermost/mattermost-plugin-msteams-meetings.
```

File: source/messaging/extending-messaging-with-integrations.md [hunk 29]
```diff
@@ -1,9 +1,111 @@
...
+
 ## Incoming Webhooks
 
 Incoming webhooks from external integrations can post messages to Mattermost in public and private channels. Learn more about setting up incoming webhooks on our [documentation page](https://docs.mattermost.com/developer/webhooks-incoming.html).
```
Comment (cwarnermm) [inline source/messaging/extending-messaging-with-integrations.md:111]: I believe these webhook links are now outdated (this one and the one below), as well as the slash command one

**Review by cwarnermm — APPROVED** [review]
Provided feedback inline, including updated URLs for links. Great job! Approving for expediency.

File: source/messaging/extending-messaging-with-integrations.md (hunk 3 above)
Comment (justinegeffen) [reply source/messaging/extending-messaging-with-integrations.md:20]: In this case we do mention them in the README but as it's maintained by us we don't link to their account directly in our docs.

Comment (cwarnermm) [reply source/messaging/extending-messaging-with-integrations.md:20]: Makes sense. Please disregard the remaining comments of the same suggestion. :)

File: source/messaging/extending-messaging-with-integrations.md (hunk 29 above)
Comment (justinegeffen) [reply source/messaging/extending-messaging-with-integrations.md:114]: They are! I'll update.

File: source/messaging/extending-messaging-with-integrations.md [hunk 30]
```diff
@@ -1,9 +1,114 @@
...
```
Comment (justinegeffen) [inline source/messaging/extending-messaging-with-integrations.md:114]:
```suggestion
Incoming webhooks from external integrations can post messages to Mattermost in Public and Private channels. Learn more about setting up incoming webhooks in our [documentation](https://developers.mattermost.com/integrate/admin-guide/admin-webhooks-incoming/).
```

File: source/messaging/extending-messaging-with-integrations.md (hunk 15 above)
Comment (justinegeffen) [reply source/messaging/extending-messaging-with-integrations.md:65]: I have no idea but it is definitely not correct. I have found the correct link.

File: source/messaging/extending-messaging-with-integrations.md [hunk 31]
```diff
@@ -1,12 +1,117 @@
...
+
+There are several ways to connect Atlassian tools to Mattermost. Take a look at the tools recommended below, or visit the Marketplace to find others.
+
+#### Confluence
+
+- Atlassian Marketplace application for Confluence and Mattermost, developed by codefortynine.
+- Send notifications about page, blogpost, question, or comment updates from your Confluence spaces to Mattermost channels.
+- Configure notifications for a specific space, and also user notifications for tasks and mentions.
+- Supports Confluence Cloud, Server and Data Center platforms.
+- Marketplace download + docs: https://marketplace.atlassian.com/apps/1215055/slack-for-confluence
```
Comment (justinegeffen) [inline source/messaging/extending-messaging-with-integrations.md:65]:
```suggestion
- Marketplace download + docs: https://marketplace.atlassian.com/apps/1222417/mattermost-connector-for-confluence
```

File: source/messaging/extending-messaging-with-integrations.md [hunk 32]
```diff
@@ -1,12 +1,117 @@
 ## Integrations
 
-Settings for integrations are accessible from the **Main Menu** by clicking the three dots at the top of the channels pane. Clicking **Integrations** opens a page where you can view and configure incoming webhooks, outgoing webhooks, and slash commands for your team. If you can't see an **Integrations** option, then your System Admin may have only given Admins access.
+Settings for integrations are accessible from the **Main Menu**. Select **Integrations** to open a page where you can view and configure incoming webhooks, outgoing webhooks, and slash commands for your team. If you can't see an **Integrations** option, then your System Admin may have only given Admins access.
 
 [Visit our app directory](https://about.mattermost.com/default-app-directory/) for dozens of open source integrations to common tools like Jira, Jenkins, GitLab, Trac, Redmine, and Bitbucket, along with interactive bot applications (Hubot, mattermost-bot), and other communication tools (Email, IRC, XMPP, Threema) that are freely available for use and customization.
```
Comment (justinegeffen) [inline source/messaging/extending-messaging-with-integrations.md:5]:
```suggestion
[Visit our Marketplace](https://mattermost.com/marketplace/) for dozens of open source integrations to common tools like Jira, Jenkins, GitLab, Trac, Redmine, and Bitbucket, along with interactive bot applications (Hubot, mattermost-bot), and other communication tools (Email, IRC, XMPP, Threema) that are freely available for use and customization.
```

File: source/messaging/extending-messaging-with-integrations.md [hunk 33]
```diff
@@ -1,12 +1,117 @@
...
 
+### Microsoft integrations
+
+There are several ways to connect your Microsoft tools to Mattermost. Take a look at the tools recommended below, or visit the Marketplace to find others.
```
Comment (justinegeffen) [inline source/messaging/extending-messaging-with-integrations.md:9]:
```suggestion
There are several ways to connect your Microsoft tools to Mattermost. Take a look at the tools recommended below, or visit the [Marketplace](https://mattermost.com/marketplace/) to find others.
```

File: source/messaging/extending-messaging-with-integrations.md [hunk 34]
```diff
@@ -1,12 +1,117 @@
...
+- Receive a daily summary of calendar events, and accept or decline new events.
+- Reflect user status as "Do Not Disturb" when in a meeting scheduled via Outlook Calendar.
+- Visit the Marketplace to install, configure, and use this integration.
+
+#### Skype for Business
+
+- Start and join voice calls, video calls, and use screen-sharing with your team members. Developed by kosgrz and maintained by Mattermost.
+- Selecting a video icon in a Mattermost channel invites team members to join a Skype for Business call, hosted using the credentials of the user who initiated the call.
+- Supports a vendor-hosted cloud solution.
+- Visit the Marketplace to install, configure, and use this integration.
```
Comment (justinegeffen) [inline source/messaging/extending-messaging-with-integrations.md:23]:
```suggestion
- Visit the [Marketplace](https://mattermost.com/marketplace/) to install, configure, and use this integration.
```

File: source/messaging/extending-messaging-with-integrations.md [hunk 35]
```diff
@@ -1,12 +1,117 @@
...
+- Source code: https://github.com/mattermost/mattermost-plugin-skype4business
+
+#### Office 365 SSO (E20)
+
+- Configure Mattermost to use your Office 365 credentials and Azure Active Directory account as a single sign-on (SSO) service, developed by Mattermost.
+- Supports Microsoft Active Directory Tenants for team creation, account creation, and Single Sign-On (SSO).
+- Receive a daily summary of calendar events, and accept or decline new events.
+- Visit the Marketplace to install, configure, and use this integration.
```
Comment (justinegeffen) [inline source/messaging/extending-messaging-with-integrations.md:31]:
```suggestion
- Visit the [Marketplace](https://mattermost.com/marketplace/) to install, configure, and use this integration.
```

File: source/messaging/extending-messaging-with-integrations.md [hunk 36]
```diff
@@ -1,12 +1,117 @@
...
+
+There are several ways to connect your CI/CD tools to Mattermost. Take a look at the tools recommended below, or visit the Marketplace to find others.
+
+#### GitHub
+
+ - Two-way integration between Mattermost and GitHub, developed by Mattermost.
+ - Get reminders on issues and pull requests that need your attention.
+ - Get notifications in Mattermost about mentions, review requests, and comments.
+ - Supports GitHub Enterprise, works with SaaS and Enterprise versions of GitHub.
+ - Visit the Marketplace to install, configure, and use this integration.
```
Comment (justinegeffen) [inline source/messaging/extending-messaging-with-integrations.md:43]:
```suggestion
 - Visit the [Marketplace](https://mattermost.com/marketplace/) to install, configure, and use this integration.
```

File: source/messaging/extending-messaging-with-integrations.md [hunk 37]
```diff
@@ -1,12 +1,117 @@
...
+ - Source code: https://github.com/mattermost/mattermost-plugin-github
+
+#### GitLab
+
+ - Two-way integration between Mattermost and GitLab, developed by Romain Maneschi, supported by Mattermost.
+ - Get reminders on issues and merge requests that need your attention.
+ - Get notifications in Mattermost about mentions, review requests, and comments.
+ - Supports SaaS and on-prem versions of GitLab.
+ - Visit the Marketplace to install, configure, and use this integration.
```
Comment (justinegeffen) [inline source/messaging/extending-messaging-with-integrations.md:52]:
```suggestion
 - Visit the [Marketplace](https://mattermost.com/marketplace/) to install, configure, and use this integration.
```

File: source/messaging/extending-messaging-with-integrations.md [hunk 38]
```diff
@@ -1,12 +1,117 @@
...
+ - Source code: https://github.com/mattermost/mattermost-plugin-gitlab
+
+### Atlassian integrations
+
+There are several ways to connect Atlassian tools to Mattermost. Take a look at the tools recommended below, or visit the Marketplace to find others.
```
Comment (justinegeffen) [inline source/messaging/extending-messaging-with-integrations.md:57]:
```suggestion
There are several ways to connect Atlassian tools to Mattermost. Take a look at the tools recommended below, or visit the [Marketplace](https://mattermost.com/marketplace/) to find others.
```

File: source/messaging/extending-messaging-with-integrations.md [hunk 39]
```diff
@@ -1,12 +1,117 @@
...
+#### Office 365 SSO (E20)
+
+- Configure Mattermost to use your Office 365 credentials and Azure Active Directory account as a single sign-on (SSO) service, developed by Mattermost.
+- Supports Microsoft Active Directory Tenants for team creation, account creation, and Single Sign-On (SSO).
+- Receive a daily summary of calendar events, and accept or decline new events.
+- Visit the Marketplace to install, configure, and use this integration.
+
+### CI/CD integrations
+
+There are several ways to connect your CI/CD tools to Mattermost. Take a look at the tools recommended below, or visit the Marketplace to find others.
```
Comment (justinegeffen) [inline source/messaging/extending-messaging-with-integrations.md:35]:
```suggestion
There are several ways to connect your CI/CD tools to Mattermost. Take a look at the tools recommended below, or visit the [Marketplace](https://mattermost.com/marketplace/) to find others.
```

File: source/messaging/extending-messaging-with-integrations.md [hunk 40]
```diff
@@ -1,12 +1,117 @@
...
+
+- Two-way integration between Mattermost and Jira, developed by Mattermost.
+- Send event notifications from your Jira projects to Mattermost channels, with full JQL filtering support.
+- Create and transition Jira issues, and attach Mattermost messages to Jira in the Mattermost user interface.
+- Supports Jira Core and Jira Software products, for Cloud, Server, and Data Center platforms.
+- Source code + documentation: https://github.com/mattermost/mattermost-plugin-jira
+
+### Productivity integrations
+
+There are several ways to connect your productivity tools to Mattermost. Take a look at the tools recommended below, or visit the Marketplace to find others.
```
Comment (justinegeffen) [inline source/messaging/extending-messaging-with-integrations.md:77]:
```suggestion
There are several ways to connect your productivity tools to Mattermost. Take a look at the tools recommended below, or visit the [Marketplace](https://mattermost.com/marketplace/) to find others.
```

File: source/messaging/extending-messaging-with-integrations.md [hunk 41]
```diff
@@ -1,12 +1,117 @@
...
+
+#### Remind Plugin
+
+ - A Mattermost plugin that sets reminders for users and channels, developed by Scott Lee Davis.
+ - Set reminders for one-time or recurring events and tasks in any Mattermost channel.
+ - Source code + docs: https://github.com/scottleedavis/mattermost-plugin-remind
+
+### Voice, video, and screensharing integrations
+
+There are several ways to work with voice, video, and screensharing in Mattermost. Take a look at the tools recommended below, or visit the Marketplace to find others.
```
Comment (justinegeffen) [inline source/messaging/extending-messaging-with-integrations.md:93]:
```suggestion
There are several ways to work with voice, video, and screensharing in Mattermost. Take a look at the tools recommended below, or visit the [Marketplace](https://mattermost.com/marketplace/) to find others.
```

**Pull request merged**